### ISSUE TITLE
Price Precision (4 Decimals)

### ISSUE THREAD

**Issue by SecurelyAeroplane** [louisnw01/lightweight-charts-python#34]
Hi!

First off, I appreciate your work.
Now regarding the issue,, is it possible the get 4 decimal prices?

![image](https://github.com/louisnw01/lightweight-charts-python/assets/138993268/25bc6385-5d75-4a2a-be9e-1c492076d951)
![image](https://github.com/louisnw01/lightweight-charts-python/assets/138993268/c5997fbc-2979-4303-98a8-c4121c73549a)

Sorry if this has been asked before!

**Comment by louisnw01** [comment]
Hi!

It actually hasn't been asked before, but this should be implemented. I will add a `precision` method within the next update.

If you want to do this with the library now, you can use the following code:

```python
precision = 4 # Change this to the number of decimals.

min_move = 1 / (10 ** precision)
chart.run_script(f'''
    {chart.id}.series.applyOptions({{
        priceFormat: {{precision: {precision}, minMove: {min_move}}}
    }})''')
```

**Comment by SecurelyAeroplane** [comment]
Worked like a charm, thanks!
![image](https://github.com/louisnw01/lightweight-charts-python/assets/138993268/1314fc4d-d1b9-4efb-8b8e-dd0e3f075aea)

**Comment by louisnw01** [comment]
Hey,

This feature has now been implemented in the latest version (1.0.14) with the `precision` method.
